apps/web/src/app/(dashboard)/admin/page.tsx

```diff
@@ -0,0 +1,114 @@
+import {
+  Archive,
+  File,
+  FileX2,
+  LucideIcon,
+  User as LucideUser,
+  Mail,
+  MailOpen,
+  PenTool,
+  Send,
+  UserPlus2,
+  UserSquare2,
+} from 'lucide-react';
+
+import { getDocsCount } from '@documenso/lib/server-only/admin/get-documents-stats';
+import { getRecipientsStats } from '@documenso/lib/server-only/admin/get-recipients-stats';
+import {
+  getUsersCount,
+  getUsersWithSubscriptionsCount,
+} from '@documenso/lib/server-only/admin/get-users-stats';
+import {
+  ReadStatus as InternalReadStatus,
+  SendStatus as InternalSendStatus,
+  SigningStatus as InternalSigningStatus,
+} from '@documenso/prisma/client';
+
+import { CardMetric } from '~/components/(dashboard)/metric-card/metric-card';
+
+type TCardData = {
```

Comment:
I think the intention of having T prefixed on a type is for inferred ZOD schemas, normal types would normally go without them

@Mythie pls correct me if wrong